modules/ROOT/pages/design-create-publish-api-raml-editor.adoc

```diff
@@ -9,11 +9,15 @@ You can use API Designer's code editor to create API specifications directly in
 Within Anypoint Platform, your user ID must be assigned the _Design Center Developer_ permission.
 
 == Procedure
-. On the *Projects* page in Design Center, click *Create* and then click *Create API Specification*.
-. In the *New API Specification* dialog, name your project and select the option *Code editor* to use the code editor. You can also choose whether the stub specification file created in the project should be in RAML or OAS. The default is to create a RAML 1.0 file; however, you can change the file to RAML 0.8 or OAS after the project is created. In fact, projects for creating API specifications can contain a mix of RAML and OAS files.
-+
-_Result:_ The code editor opens. The editor is divided into three panels:
+
+. Create an API-specification project that you can work in.
+ .. On the *Projects* page in Design Center, click *Create* and then click *Create API Specification*.
+ .. In the *New Specification* section of the dialog that is opened, name your project and select the option *Code editor* to use the code editor.
```

Comment:
can we change "that is opened" to "that opens" ? NaBD.

Reply:
"Open" is a transitive verb.